File: lib/Screens/payment_and_invoice/models/verification_remark_model.dart
class VerificationRemarkResponse {
  String? status;
  String? message;
  List<VerificationRemarkOutput>? output;

  VerificationRemarkResponse({this.status, this.message, this.output});

  VerificationRemarkResponse.fromJson(Map<String, dynamic> json) {
    status = json['status'];
    message = json['message'];
    if (json['output'] != null) {
      output = <VerificationRemarkOutput>[];
      json['output'].forEach((v) {
        output!.add(VerificationRemarkOutput.fromJson(v));
      });
    }
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['status'] = status;
    data['message'] = message;
    if (output != null) {
      data['output'] = output!.map((v) => v.toJson()).toList();
    }
    return data;
  }
}

class VerificationRemarkOutput {
  int? verificationRemarkID;
  String? verificationRemark;
  int? rankID;

  VerificationRemarkOutput({
    this.verificationRemarkID,
    this.verificationRemark,
    this.rankID,
  });

  VerificationRemarkOutput.fromJson(Map<String, dynamic> json) {
    verificationRemarkID = json['VerificationRemarkID'];
    verificationRemark = json['VerificationRemark'];
    rankID = json['RankID'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['VerificationRemarkID'] = verificationRemarkID;
    data['VerificationRemark'] = verificationRemark;
    data['RankID'] = rankID;
    return data;
  }
}

File: S2T Operational - After - 9.82/S2T Operational - After - 9.82/app/src/main/java/com/myhindlab/abkat/activities/payout/model/VerificationRemarkModel.java
package com.myhindlab.abkat.activities.payout.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


import java.util.List;

public class VerificationRemarkModel {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("output")
    @Expose
    private List<Output> output;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Output> getOutput() {
        return output;
    }

    public void setOutput(List<Output> output) {
        this.output = output;
    }


    public class Output {

        @SerializedName("VerificationRemarkID")
        @Expose
        private Integer verificationRemarkID;
        @SerializedName("VerificationRemark")
        @Expose
        private String verificationRemark;
        @SerializedName("RankID")
        @Expose
        private Integer rankID;

        public Integer getVerificationRemarkID() {
            return verificationRemarkID;
        }

        public void setVerificationRemarkID(Integer verificationRemarkID) {
            this.verificationRemarkID = verificationRemarkID;
        }

        public String getVerificationRemark() {
            return verificationRemark;
        }

        public void setVerificationRemark(String verificationRemark) {
            this.verificationRemark = verificationRemark;
        }

        public Integer getRankID() {
            return rankID;
        }

        public void setRankID(Integer rankID) {
            this.rankID = rankID;
        }

    }
}